### PR TITLE
add WStrings message

### PR DESCRIPTION
**Pull request by dirk-thomas**
Connect to ros2/rosidl#352.

### CODE REVIEW

File: msg/WStrings.msg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+wstring wstring_value
+#wstring wstring_value_default1 "Hello world!"
```
Comment (jacobperron) [inline msg/WStrings.msg:2]: Maybe add a TODO explaining why these are commented out.

**Review by jacobperron — APPROVED** [review]

File: msg/WStrings.msg (hunk 1 above)
Comment (dirk-thomas) [reply msg/WStrings.msg:2]: I will follow on these right away trying to enable all of them...

**Comment by adamsj-ros** [comment]
why is this part of test_msgs and not a built-in type? 

https://index.ros.org/doc/ros2/Concepts/About-ROS-Interfaces/#field-types

**Comment by jacobperron** [comment]
> why is this part of test_msgs and not a built-in type?

`wstring` **is** a built-in type. It was added in ROS Dashing. Here's the design doc: http://design.ros2.org/articles/wide_strings.html

This PR introduces a message for testing the built-in type.

It looks like the ROS Index page is missing info on `wstring`. I've opened a ticket for adding it: https://github.com/ros2/ros2_documentation/issues/620

**Comment by adamsj-ros** [comment]
Shouldn't it be in [std_msgs](https://github.com/ros2/common_interfaces/tree/master/std_msgs/msg) too to be considered a built-in type?

**Comment by dirk-thomas** [comment]
@adamsj-oci there is a difference between built-in types and messages. A built-in type can be used as a field type in a message but is itself not a message.

**Comment by adamsj-ros** [comment]
@dirk-thomas That makes sense but at the same time it seems logical to have a `Wstring.msg` in a similar way we have `String.msg` if nothing else to support special characters in non-English languages.

**Comment by jacobperron** [comment]
I agree that it makes sense to add a `WString.msg` alongside the existing `String.msg`. However, note there's an open ticket proposing to move many of the messages in `std_msgs` to a different package. I'm not sure if the plan is to move `String.msg` as well.

https://github.com/ros2/common_interfaces/issues/89

**Comment by dirk-thomas** [comment]
> it seems logical to have a `Wstring.msg` in a similar way we have `String.msg` if nothing else to support special characters in non-English languages. 

This package as the name indicates is only for tests. You should create your own message definition to give the interface semantic meaning - for the same reason why you shouldn't use `std_msgs` in your packages.

**Pull request merged**